src/python/py/models/builder.py

```diff
@@ -2799,7 +2963,16 @@ def create_model(model_name, input_path, output_dir, precision, execution_provid
     extra_kwargs = {} if os.path.isdir(input_path) else {"cache_dir": cache_dir}
     hf_name = input_path if os.path.isdir(input_path) else model_name
     hf_token = parse_hf_token(extra_options.get("hf_token", "true"))
-    config = AutoConfig.from_pretrained(hf_name, token=hf_token, trust_remote_code=True, **extra_kwargs)
+
+    is_peft = os.path.isdir(adapter_path) and "adapter_config.json" in os.listdir(adapter_path)
```

Comment:
bring this discussion online: the check for `isdir` and `adapter_config.json` requires that the `adapter_path` is local. Is this check required? 
I think the `isdir` check in line 2965 for `input_path` is present to differentiate between a local hf checkpoint (dir) and a local gguf checkpoint (file). 

Since we don't have such issues for adapter_path, checking for `not adapter_path` might be enough? this way it can also support adapters on hf-hub.

Reply:
Removed the adapter_path arg so no longer required